collect_app/src/main/java/org/odk/collect/android/preferences/AdminPreferencesFragment.java

```diff
@@ -149,7 +150,9 @@ public void onClick(DialogInterface dialog, int which) {
         }
 
         if (fragment != null) {
-            getActivity().getFragmentManager().beginTransaction()
+            getActivity().getFragmentManager()
+                    .beginTransaction()
+                    .setCustomAnimations(R.animator.push_left_in_fragment,R.animator.push_left_out_fragment,R.animator.push_right_in_fragment,R.animator.push_right_out_fragment)
```

Comment:
Add a space after each comma.  Since this line is so long, the four items should also be on separate lines.